packages/contracts/contracts/TransactionManager.sol

```diff
@@ -204,9 +205,7 @@ contract TransactionManager is ReentrancyGuard, ProposedOwnable, ITransactionMan
     require(routerBalance >= amount, "#RL:008");
 
     // Update router balances
-    unchecked {
-      routerBalances[msg.sender][assetId] = routerBalance - amount;
-    }
+    routerBalances[msg.sender][assetId] = routerBalance - amount;
```

Comment:
If we're going to leave this change for both chains we should probably remove the require above